### PR TITLE
Mobile menu

### PR DESCRIPTION
**Pull request by Shakir-Hussain12**
### This Merge consists of the following additions
---
- Functional Hamburger Button for the Mobile version/
- Underlined nav items for the desktop version.
- Linter & ESLint checks for JS files.

### CODE REVIEW

File: script.js [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+const body = document.querySelector('body');
+const li = document.querySelectorAll('.page-flow > li');
+const x = window.matchMedia('(max-width: 768px)');
+const sec = document.querySelectorAll('*');
+let id;
+
+function myFunction(x) {
+  // If media query matches
+  if (x.matches) {
+    x.addListener(myFunction);
```
Comment (zieeco) [inline script.js]: - [x]  Hey, this chained statement is preventing your mobile menu from showing up, I fixed it from my end and it worked as expected. kindly double-check the statement closely to fix this issue.
- [x] I notice your background color is not the same as in the Figma design. Please, kindly ensure you've used the correct background to match the design after fixing the issue stated above


*Figma*
Screenshot from 2022-12-19 20-48-20


*yours*
Screenshot from 2022-12-19 20-47-39

**Review by zieeco — CHANGES_REQUESTED** [review]
Hi @Team,


Great job so far! 👏 
You've done well, however, there are some issues that you still need to work on to go to the next project, but you are almost there!


## To highlight 🎯

- No linter errors.✔️
- Nice design. ✔️
- Descriptive PR. ✔️


## Required Changes ♻️
_Check the comments under the review._





## Optional suggestions
- N/A

**_Every comment with the **[OPTIONAL]** prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._**


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: script.js (hunk 1 above)
Comment (Shakir-Hussain12) [reply script.js]: @zieeco I have fixed it, but using another method. Can you please tell me how you did it with this existing one?
I tried a couple of things, but couldn't reach a solution as it was always failing some test case.

**Review by muneebulrehman — APPROVED** [review]
Hi @Shakir-Hussain12 , 

Your project is complete!  There is nothing else to say other than... it's time to merge it 🥳
Congratulations! 🎉 

# Status: Approved✔️

## Highlights
- No linter errors
- Descriptive PR
- Good follow-up with the previous reviewer.

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**